c++/src/kj/test.c++

```diff
@@ -346,6 +346,8 @@ public:
     if (passCount > 0) write(GREEN, kj::str(passCount, " test(s) passed"), "");
     if (failCount > 0) write(RED, kj::str(failCount, " test(s) failed"), "");
     context.exit();
+
+    return true;
```

Comment:
Hmm, which compiler warned about this missing return? `context.exit()` is marked with the `noreturn` attribute, which should prevent such warnings.

Does it work if you put `KJ_UNREACHABLE` here instead of `return true`? I generally prefer that as it is more self-documenting.